slam-libraries/viam-cartographer/README.md

```diff
@@ -36,21 +46,21 @@ brew upgrade
 # Install dependencies
 brew install abseil boost ceres-solver protobuf ninja cairo googletest lua@5.3
 brew link lua@5.3
+brew link protobuf
+brew install openssl eigen gflags glog suite-sparse sphinx-doc pcl
```

Comment:
Sorry - what I meant was the opposite: Rather than listing the dependencies manually, the user could just run `./scripts/setup_cartographer_macos.sh` here. I'd therefore replace this with:

```bash
# Install dependencies
./scripts/setup_cartographer_macos.sh
```

Reply:
i added a block above for automatic installation. So either a user can use the auto install script, or get dependencies manually if they have specific versions of these packages that they want to use

Reply:
tweaked wording abit to make it clear this is for manual installation, and we want to keep these because it shows users what is actually getting installed on their systems